content/en/blog/releases/v0.8.0-release-notes.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Build v0.8.0"
+date: 2022-02-02T10:29:59-04:00
+draft: false
+---
+
+Shipwright Build v0.8.0 can be found on GitHub at [shipwright-io/build v0.8.0](https://github.com/shipwright-io/build/releases/tag/v0.8.0). The CLI is available at [shipwright-io/cli v0.8.0](https://github.com/shipwright-io/cli/releases/tag/v0.8.0).
+
+## Upgrade Instructions
+
+1. Install v0.8.0 using the release YAML manifest:
+
+   ```bash
+   $ kubectl apply -f https://github.com/shipwright-io/build/releases/download/v0.8.0/release.yaml
+   ```
+
+2. (Optionally) Install the sample build strategies using the YAML manifest:
+
+   ```bash
+   $ kubectl apply -f https://github.com/shipwright-io/build/releases/download/v0.8.0/sample-strategies.yaml
+   ```
+
```

Comment:
I will submit a follow-up PR with instructions on how to update the operator, once the new version is approved and merged into operatorhub.io. See https://github.com/k8s-operatorhub/community-operators/pull/697